src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonFinderFeatures.kt

```diff
@@ -149,27 +150,30 @@ class DungeonFinderFeatures {
     @SubscribeEvent
     fun onItemTooltip(event: LorenzToolTipEvent) {
         if (!LorenzUtils.inSkyBlock) return
-        if (!config.coloredClassLevel) return
+
+        val coloredClass = config.coloredClassLevel
```

Comment:
I feel this isnt needed here and you can just do config.coloredClassLevel further down

Reply:
This was fixed not sure why it didn't push the code though.

Reply:
oh nvm I just understood what you meant